test/scrabble_test.rb

```diff
@@ -6,8 +6,19 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_that_score_is_insensitive_to_case
+    assert_equal 1, Scrabble.new.score("A")
+    assert_equal 4, Scrabble.new.score("f")
+  end
```

Comment:
Like that we're testing for uppercase here. Testing for lowercase seems duplicative.